pkg/cmd/gc/activities/activities.go

```diff
@@ -108,6 +108,8 @@ func NewCmdGCActivities() (*cobra.Command, *Options) {
 	cmd.Flags().DurationVarP(&o.ReleaseAgeLimit, "release-age", "r", time.Hour*24*30, "Maximum age to keep PipelineActivities for Releases")
 	cmd.Flags().DurationVarP(&o.PipelineRunAgeLimit, "pipelinerun-age", "", time.Hour*12, "Maximum age to keep completed PipelineRuns for all pipelines")
 	cmd.Flags().DurationVarP(&o.ProwJobAgeLimit, "prowjob-age", "", time.Hour*24*7, "Maximum age to keep completed ProwJobs for all pipelines")
+	cmd.Flags().BoolVarP(&o.AllNamespaces, "all-namespaces", "", false, "Work in all namespaces (requires a ClusterRole)")
```

Comment:
need to run `make docs` to update the documentation.

Reply:
Sure will do